cypress/e2e/smoke/landing.cy.js

```diff
@@ -0,0 +1,35 @@
+describe('Landing page', () => {
+  it('redirects to welcome page', () => {
+    cy.visit('/')
+
+    cy.url().should('include', '/welcome')
+  })
+
+  it('should require accept "Updates" cookies to display Beamer', () => {
+    cy.visit('/')
+
+    // Way to select the cookies banner without an id
+    cy.contains('We use cookies to provide').parent('div').should('be.visible')
```

Comment:
Is there a reason we are not checking for the existence of the content directly without going through the parent element? i.e. `cy.contains('We use cookies to provide').should('be.visible')`

Reply:
fair enough. I think I blindly replaced the old code 👍